src/Hypergrid/scrolling.ts

```diff
@@ -277,36 +277,43 @@ exports.mixin = {
         this.resizeScrollbars();
     },
 
-    resizeScrollbars: function() {
+    resizeScrollbars: function () {
         this.sbHScroller.shortenBy(this.sbVScroller).resize();
-        //this.sbVScroller.shortenBy(this.sbHScroller);
-        this.sbVScroller.resize();
+        this.sbVScroller.shortenBy(this.sbHScroller).resize();
     },
 
     /**
      * @memberOf Hypergrid#
      * @this {Hypergrid}
      * @desc Scroll values have changed, we've been notified.
      */
-    setVScrollbarValues: function(max) {
+    setVScrollbarValues: function (max, containerSize) {
+        // Set the scroll range, which by default resets the contentSize.
         this.sbVScroller.range = {
             min: 0,
             max: max
         };
+        // Redefine the sizes.
+        this.sbVScroller.contentSize = max + containerSize;
```

Comment:
Key change, related to another code change I'll point out. 
This sets the content size (all rows), which is incorrectly set by the range setter implementation.
It applies the number of rows that can be scrolled, plus the rows visible.